tests/integration/tier0/test-check-rollback-handling/test_check_rollback_handling.py

```diff
@@ -0,0 +1,81 @@
+import os
+import platform
+
+from envparse import env
+
+
+def test_proper_rhsm_clean_up(shell, convert2rhel):
+    """Test that c2r does not remove usermod, rhn-setup and os-release during rollback.
+    Also checks that the system was successfully unregistered.
+    """
+
+    # Ensure usermode and rhn-setup packages are presented
+    assert shell("yum install -y usermode rhn-setup").returncode == 0
+
+    # run c2r until subscribing the system and then emulate pressing Ctrl + C
```

Comment:
@kokesak To be honest, I'm not sure about this comment saying 'emulate pressing Ctrl + C' since there is no emulation as such.

Reply:
Good catch, it's from the old test. Please remove the ctrl+c part

Reply:
Sure, thanks!